mods/ra/rules/palettes.yaml

```diff
@@ -1,7 +1,8 @@
 ^Palettes:
-	PlayerPaletteFromCurrentTileset:
+	PaletteFromFile@player:
 		Name: player
-		ShadowIndex: 3,4
```

Comment:
I just didn't know this served a purpose, since only index 4 is used for shadows in the original.